client/app/admin/create-project/create-project.controller.js

```diff
@@ -11,9 +11,27 @@
 
     function createProjectController(mapService, drawService, projectService) {
         var vm = this;
+
+        // Wizard variables
         vm.currentStep = '';
+        vm.taskGrid = false;
+        vm.taskArbitrary = false;
+
+        // AOI variables
         vm.AOIValid = true;
         vm.AOIValidationMessage = '';
+        vm.AOI = null;
+
+        // Creating grid variables
+        vm.sizeOfTasks = 0; 
```

Comment:
constants do exist in modern JS land but it is not supported by all browsers. So I've used the convention to put them in uppercase.